src/main/java/taxi/Application.java

```diff
@@ -31,54 +29,20 @@ public static void main(String[] args) {
         w221 = new Car("w221", mercedesManufacturer);
         q8 = new Car("q8", audiManufacturer);
         testManufacturerService();
-        testCarService();
-        testDriverService();
-    }
-
-    public static void testCarService() {
-        CarService carService =
-                (CarService) injector.getInstance(CarService.class);
-        carService.create(x5);
-        carService.create(w221);
-        carService.create(q8);
-        carService.getAll();
-        carService.addDriverToCar(vasya, x5);
-        carService.addDriverToCar(petrovich, w221);
-        carService.addDriverToCar(ivan, q8);
-        System.out.println(carService.getAll());
-        System.out.println(carService.getAllByDriver(2L));
-        carService.removeDriverFromCar(vasya, x5);
-        carService.delete(1L);
-        System.out.println(carService.getAll());
-    }
-
-    public static void testDriverService() {
-        DriverService driverService =
-                (DriverService) injector.getInstance(DriverService.class);
-        driverService.create(vasya);
-        driverService.create(ivan);
-        driverService.create(petrovich);
-        System.out.println(driverService.getAll());
-        driverService.get(1L);
-        driverService.delete(1L);
-        System.out.println(driverService.getAll());
-        driverService.update(vasya);
-        System.out.println(driverService.getAll());
     }
 
     public static void testManufacturerService() {
         ManufacturerService manufacturerService =
                 (ManufacturerService) injector.getInstance(ManufacturerService.class);
-
         manufacturerService.create(bmwManufacturer);
         manufacturerService.create(mercedesManufacturer);
         manufacturerService.create(audiManufacturer);
         System.out.println(manufacturerService.getAll());
-        Manufacturer updatedAudi = manufacturerService.get(1L);
+        Manufacturer updatedAudi = manufacturerService.get(3L);
```

Comment:
Why we get by id = 3 and delete by id = 4? I think we should create the manufacturer, get its id. Then we will pass this received id to method get and to method delete.